odometry_conversion/launch/tracking_camera_sim.launch

```diff
@@ -5,5 +5,6 @@
     <param name="in_sensor_frame" value="tracking_camera_link"/>
```

Comment:
I think `in_sensor_frame` should be `tracking_camera_pose_frame` (which is identical to `tracking_camera_link`, but is the one also mentioned in the odom message) .

Reply:
Yes, but I think during cleaning, we removed it and is no longer a frame. Maybe I can change it in the tracking camera as well. https://github.com/ETHZ-RobotX/smb_common/blob/master/smb_description/urdf/realsense_t265.urdf.xacro

Reply:
Done